test/libsolidity/InlineAssembly.cpp

```diff
@@ -429,6 +429,16 @@ BOOST_AUTO_TEST_CASE(revert)
 	BOOST_CHECK(successAssemble("{ revert(0, 0) }"));
 }
 
+BOOST_AUTO_TEST_CASE(returndatasize)
+{
+	BOOST_CHECK(successAssemble("{ let r := returndatasize }"));
```

Comment:
My goal is to catch many bugs with a smaller number of test cases.  For that, diversity is important.